tests/Directory.Build.props

```diff
@@ -15,6 +15,8 @@
 
     <!-- This is useful for local runs -->
     <VSTestResultsDirectory>$(ArtifactsTestResultsDir)</VSTestResultsDirectory>
+
+    <OutputType Condition="$(MSBuildProjectName.EndsWith('.Tests'))">Exe</OutputType>
```

Comment:
Any reason not go all the way and use `<Project Sdk="MSTest.Sdk/x.y.z">` which, essentially, means the tests are run as exe?

Reply:
This repo is xUnit, not MSTest.